### PR TITLE
Add condensed Promotions for V2

### PR DESCRIPTION
**Pull request by rahinisrinivas**

### CODE REVIEW

File: docs/carts-orders/carts/cart-management/get-a-cart.md [hunk 1]
```diff
@@ -249,42 +249,47 @@ curl -X GET https://useast.api.elasticpath.com/v2/carts/:id?include=promotions \
 
 ## Response Example
 
+The following response example shows condensed promotions for cart discounts within the included object. 
+
+When you create a cart discount rule promotion, an item discount rule promotion and a standard cart discount promotion, aand apply these discounts to a cart with eligible items, then upon calling `GET {{baseUrl}}/carts/{{cartID}}?include=promotions` endpoint, the cart relationship object will have the promotion item. Aditionally, the `included` object will contain condensed promotions for the cart discounts.
```
Comment (kaiyuancheng) [inline docs/carts-orders/carts/cart-management/get-a-cart.md:254]: typo `aand`

File: docs/rule-promotions/overview.md [hunk 2]
```diff
@@ -514,6 +514,19 @@ See the following response example:
 }
 ``` 
 
+## Condensed Promotion
+
+Condensed Promotion generally refers to the process of simplifying or aggregating multiple promotional rules into a single, more manageable rule. A condensed promotion is essentially an object that contains specific meta information about a promotion applied to the cart or order.
```
Comment (kaiyuancheng) [inline docs/rule-promotions/overview.md:519]: The sentence `Condensed Promotion generally refers to the process of simplifying or aggregating multiple promotional rules into a single, more manageable rule. ` 
doesn't seem to fit here.

File: docs/carts-orders/carts/cart-management/get-a-cart.md (hunk 1 above)
Comment (rahinisrinivas) [reply docs/carts-orders/carts/cart-management/get-a-cart.md]: done

**Review by kaiyuancheng — APPROVED** [review]

File: docs/carts-orders/get-a-cart.md [hunk 3]
```diff
@@ -249,42 +249,47 @@ curl -X GET https://useast.api.elasticpath.com/v2/carts/:id?include=promotions \
 
 ## Response Example
 
+The following response example shows condensed promotions for cart discounts within the included object. 
+
+When you create a cart discount rule promotion, an item discount rule promotion and a standard cart discount promotion, and apply these discounts to a cart with eligible items, then upon calling `GET {{baseUrl}}/carts/{{cartID}}?include=promotions` endpoint, the cart relationship object will have the promotion item. Aditionally, the `included` object will contain condensed promotions for the cart discounts.
```
Comment (johughes1204) [inline docs/carts-orders/get-a-cart.md:254]:
```suggestion
When you create a cart discount rule promotion, an item discount rule promotion and a standard cart discount promotion, and apply these discounts to a cart with eligible items, then upon calling `GET {{baseUrl}}/carts/{{cartID}}?include=promotions` endpoint, the cart relationship object has the promotion item. Aditionally, the `included` object contains condensed promotions for the cart discounts.
```

File: docs/rule-promotions/overview.md [hunk 4]
```diff
@@ -521,6 +521,19 @@ See the following response example:
 }
 ``` 
 
+## Condensed Promotion
+
+A condensed promotion is essentially an object that contains specific meta information about a promotion applied to the cart or order. 
+
+Condensed Promotion can be retrived from the following endpoints:
```
Comment (johughes1204) [inline docs/rule-promotions/overview.md:528]:
```suggestion
Condensed promotions can be retrived from the following endpoints:
```

File: docs/rule-promotions/overview.md [hunk 5]
```diff
@@ -521,6 +521,19 @@ See the following response example:
...
+
+ - [Get a Cart](/docs/carts-orders/get-a-cart#get-include-promotions): This endpoint displays cart-level condensed promotions within the included object. If `?include=items` is specified, the included object will also contain cart item-level promotions. The relationships field at the cart level will have a promotions section only when `?include=promotions` is specified.
```
Comment (johughes1204) [inline docs/rule-promotions/overview.md:530]:
```suggestion
 - [Get a Cart](/docs/carts-orders/get-a-cart#get-include-promotions): This endpoint displays cart-level condensed promotions within the included object. If `?include=items` is specified, the included object also contains cart item-level promotions. The relationships field at the cart level has a promotions section only when `?include=promotions` is specified.
```

**Review by johughes1204 — APPROVED** [review]

**Pull request merged**